stacks/java-openliberty-gradle/devfile.yaml

```diff
@@ -45,7 +45,7 @@ variables:
 components:
   - name: dev
     container:
-      image: icr.io/appcafe/open-liberty-devfile-stack:{{liberty-version}}-gradle
+      image: icr.io/appcafe/open-liberty-devfile-stack:22.0.0.1-gradle
```

Comment:
should also remove the global variable

Reply:
I've kept the global variable as it is still used on the stack in other places. I think upon reviewing the renovate bot PR, the reviewer should update also the value of the variable in order to get test passed (in case there is an update for this image)